.eslintrc

```diff
@@ -297,7 +297,7 @@
     // Disallow unreachable statements after a return, throw, continue, or break
     // statement.
     "no-unreachable": 2,
-    // Disallow global and local variables that aren't used, but allow unused function arguments.
+    // Disallow global and local variables that arent used, but allow unused function arguments.
```

Comment:
The syntax highlighting with apostrophe's is a pet peeve of mine. I'm not sure which is worse, breaking the highlighting or using bad english. Feel free to overrule this decision :)